docker/service/Dockerfile

```diff
@@ -20,5 +20,5 @@ COPY --from=package /home/gradle/service/build/libs/gp2gp.jar /app/gp2gp.jar
 
 USER 65534
 
-ENTRYPOINT ["java", "-cp", "/app/gp2gp.jar", "-Dloader.main=uk.nhs.adaptors.gp2gp.Gp2gpApplication", "org.springframework.boot.loader.PropertiesLauncher"]
+ENTRYPOINT ["java", "-XX:+UseG1GC", "-cp", "/app/gp2gp.jar", "-Dloader.main=uk.nhs.adaptors.gp2gp.Gp2gpApplication", "org.springframework.boot.loader.PropertiesLauncher"]
```

Comment:
Hmm, [from my reading](https://learn.microsoft.com/en-us/java/openjdk/reasons-to-move-to-java-11#garbage-collection-10) looks like this is unnecessary?

Reply:
by setting up this clearly we want to make sure that we are using G1, which is a better GC compared to the other ones. 
Sometimes JVM opt in for a different GC, like a Serial GC etc

Reply:
G1 is recommended for application with large datasets. It has become the standard GC in recent years for Java due to its excellent performance. It should be enabled in GP2GP by default (as long as there are more than 1 vCPU) I just want to reinforce that by providing the flag. Here is some brief information about https://en.wikipedia.org/wiki/Garbage-first_collector from wiki.

Reply:
@adrianclay I've decided to remove G1GC flag for now since the memory issues for NIAD-2348 was fixed by adjusting the config but still I will be monitoring the performance aspect of GP2GP to see if that flag should be re-introduced in the future.